From f742f9b57faf11018b590badbf05b549dcf9c9be Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:46:59 -0400
Subject: [x86] pci: Remove unnecessary local variable initialization

Message-id: <20140806034658.1438.79127.stgit@gir.stowe>
Patchwork-id: 87436
O-Subject: [RHEL7.1 PATCH 27/69] x86/PCI: Remove unnecessary local variable initialization
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 8928d5a66df1428fd215e611088822d0bec08f58

commit 8928d5a66df1428fd215e611088822d0bec08f58
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Jan 24 10:41:11 2014 -0700

    x86/PCI: Remove unnecessary local variable initialization

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/pci/acpi.c b/arch/x86/pci/acpi.c
index 53ba77a..efee6fc 100644
--- a/arch/x86/pci/acpi.c
+++ b/arch/x86/pci/acpi.c
@@ -473,11 +473,11 @@ probe_pci_root_info(struct pci_root_info *info, struct acpi_device *device,
 struct pci_bus *pci_acpi_scan_root(struct acpi_pci_root *root)
 {
  struct acpi_device *device = root->device;
- struct pci_root_info *info = NULL;
+ struct pci_root_info *info;
  int domain = root->segment;
  int busnum = root->secondary.start;
  LIST_HEAD(resources);
- struct pci_bus *bus = NULL;
+ struct pci_bus *bus;
  struct pci_sysdata *sd;
  int node;
 
-- 
1.7.1